ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -339,6 +339,41 @@ public struct Avatar: View {
                           with: windowProvider)
     }
 
+    /// Calculates the size of the avatar
```

Comment:
Can we move this func to be an internal member of the avatar state (MSFAvatarStateImpl)?